Assets/Character/Scripts/Consumable1.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Consumable1 : MonoBehaviour {
```

Comment:
inherit from consumable

Reply:
i know this is an example, but a proper name would be better. HealPotionConsumable?

Reply:
ah this wasn't even intended to be an example actually, but I'll make it one